library/src/main/java/com/pokegoapi/api/inventory/CandyJar.java

```diff
@@ -84,10 +63,11 @@ public void removeCandy(PokemonFamilyId family, int amount) {
 	 * @return number of candies in jar
 	 */
 	public int getCandies(PokemonFamilyId family) {
-		if (candies.containsKey(family)) {
-			return this.candies.get(family);
-		} else {
-			return 0;
-		}
+		Integer val = candies.putIfAbsent(family, 0);
+
+		if (val == null)
```

Comment:
With a putIfAbsent val will never be null, actually, val should be an int. This code can be a one liner